scripts/functions/gemset

```diff
@@ -181,7 +181,7 @@ gem_install_force()
   else install_params+=( --no-ri --no-rdoc )
   fi
 
-  install_params+=( $rvm_gem_options )
+  install_params+=( $(echo "$rvm_gem_options" | __rvm_sed -E 's/--no-(document|ri|rdoc)([[:space:]]+|$)//g') )
```

Comment:
try:
```bash
for __gem_option in $rvm_gem_options
do
  case $__gem_option
    (--no-ri|--no-rdoc|--no-document) true ;;
    (*) install_params+=( $__gem_option )
 esac
done
```

Reply:
Thanks, that's much nicer than my solution!

As long as none of the `rvm_gem_options` contain spaces your solution should work fine.

Reply:
Rebased on master and pushed new version with proposed case statement.